ush/templates/FV3LAM_wflow.xml

```diff
@@ -1517,6 +1521,46 @@ MODULES_RUN_TASK_FP script.
   </metatask>
 {%- endif %}
 
+{%- endif %}
+
+{%- if do_ensemble and do_ens_graphics %}
```

Comment:
Is there an option to turn off the regular graphics when running ensemble graphics?